src/transformers/pipelines/text2text_generation.py

```diff
@@ -185,6 +185,10 @@ def _forward(self, model_inputs, **generate_kwargs):
         generate_kwargs["max_length"] = generate_kwargs.get("max_length", self.model.config.max_length)
         self.check_inputs(input_length, generate_kwargs["min_length"], generate_kwargs["max_length"])
         output_ids = self.model.generate(**model_inputs, **generate_kwargs)
+        if generate_kwargs.get("return_dict_in_generate"):
+            # Instance check of type unions do not work until Python 3.10, e.g. `isinstance(generated_sequence, GenerateOutput):`
+            # https://peps.python.org/pep-0604/#isinstance-and-issubclass
```

Comment:
This comment is unnecessary imo. The condition you're looking at is the correct one.

Reply:
done!